front-end-node/search/SearchOverrides.js

```diff
@@ -0,0 +1,121 @@
+/*jshint browser:true, nonew:false*/
+/*global WebInspector:true, InspectorFrontendHost:true, InspectorFrontendHostAPI:true*/
+
+(function() {
+  var createSearchRegex = function(query, caseSensitive, isRegex)
+  {
+    var regexFlags = caseSensitive ? "g" : "gi";
+    var regexObject;
+
+    if (isRegex) {
+      try {
+        regexObject = new RegExp('^.*?'+query+'.*?$|^.*?'+query+'.*?\n|\n.*?'+query+'.*?\n|\n.*?'+query+'.*?$', regexFlags);
+      } catch (e) {
+        // Silent catch.
```

Comment:
@yorkie , I'm not sure that this is a best time to review internal structure of this pr. At current step we need to decide how to avoid caching. Read my last comments.

Relative to your question:
```
new RegExp('abc[')
```